### PR TITLE
Rename "handyreadgenotyper" to "amplicontyper"

### PR DESCRIPTION
**Pull request by AntonS-bio**
The original name turned out very unwieldy. AmpliconTyper has no obvious conflicts and reasonably conveys the purpose of the package.

### CODE REVIEW

File: recipes/amplicontyper/meta.yaml [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   run_exports:
-      - {{ pin_subpackage('handyreadgenotyper', max_pin="x.x") }}  
+      - {{ pin_subpackage('amplicontyper', max_pin="x.x") }}  
   number: 0
 
 requirements:
+  build:
+    - setuptools
```
Comment (martin-g) [inline recipes/amplicontyper/meta.yaml:21]: Are you sure this is needed?

Comment (AntonS-bio) [reply recipes/amplicontyper/meta.yaml:21]: Oddly yes, I didn't have it in previous releases and "conda-build" worked fine, but not this time. I've tried a lot of things and this is the only one removed the "module not found error",

Comment (martin-g) [reply recipes/amplicontyper/meta.yaml:21]: I think the one in `host` should be enough.
Let me build without the one in `build` and if it fails I will revert it!

**Review by martin-g — APPROVED** [review]

File: recipes/amplicontyper/meta.yaml (hunk 1 above)
Comment (AntonS-bio) [reply recipes/amplicontyper/meta.yaml]: You are right, the one in host is enough. I've updated meta.yaml for future releases. Thanks for checking.

**Pull request merged**